### ISSUE TITLE
Does not Like anymore

### ISSUE THREAD

**Issue by Fraeksn** [Taknok/youtube-auto-like#89]
**Expected Behavior**
Auto-Likes Videos from Creators that i Subscribed to.

**Actual Behavior**
It does nothing.
Changed Settings to "All Videos" and "Instant Like"/ect... still nothing

**Logs**
[logs.zip](https://github.com/Taknok/youtube-auto-like/files/13466808/logs.zip)
There is only liker-paper.js insteat of liker-material.js

**Specification**
 - OS: Windows 11 (Version 22H2)
 - Browser version: Firefox 120.0
 - Plugin version: 3.8.1
 - YouTube Premium: no
 - Youtube Player: Theater
 - Youtube Country: Germany

**Comment by MaeharaProjekt** [comment]
I am also experiencing this issue but the only notable differences from @Fraeksn's report are:

OS: Windows 10 (Build 19045)
Country: USA

All other specs listed by @Fraeksn are the same, as I write this comment.

Troubleshooting: 
I attempted the following actions but these did not resolve the issue,

- Auto-like does not trigger whether Subscription Notifications are set to All, Personalized, or None.
- I removed the extension, restarted Firefox, and then added the extension as if new, but Auto-like would not work.
- I also attempted the following with no improvement in current behavior:
    1. Unsubscribed from a channel.
    2. Cleared cache.
    3. Fully exited and re-launched Firefox.
    4. Re-subscribed to the same channel.
    5. Opened a video that I had not previously marked with "Like."

I don't know if this helps but I hope my addition does not muddy the waters.
